### ISSUE TITLE
Export to a standard format

### ISSUE THREAD

**Issue by hernanparra** [cmoog/vscode-sql-notebook#23]
Export a notebook with results as HTML or RICHTEXT or something I can send by email to a non-technical user.

**Comment by cmoog** [comment]
If I understand this correctly, the goal would be to share a document with a non-technical user that contains the query *and* its associated output?

I like the idea of a plain/rich text export with the query output formatted as plaintext tables underneath the queries.

**Comment by hernanparra** [comment]
Exactly. Give the user something he can read and if needed copy & paste one result grid into a spreedsheet to work on it.

**Comment by bucweat** [comment]
Gonna throw this out there…if you used markdown it could be rendered to pretty much anything else using Pandoc. I believe this is how other “notebook” apps achieve the various output formats.
